src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/AbstractHandlerProvider.cs

```diff
@@ -2,6 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// This is consumed as 'generated' code in a source package and therefore requires an explicit nullable enable
+#nullable enable
```

Comment:
we ship an .editorconfig in the package that marks all the code in here as generated code.  This ensures that analyzers do not report diagnostics for these files when included as a source package (compiler warnings are still reported - unfortunately there is no way to disable that).

And generated code requires explicit nullable defines in order to use nullable annotations.